glean-core/ios/Glean/Glean.swift

```diff
@@ -36,7 +36,7 @@ class OnGleanEventsImpl: OnGleanEvents {
 
     func triggerUpload() {
         // If uploading is disabled, we need to send the deletion-request ping
-        HttpPingUploader.launch(configuration: self.glean.configuration!)
+        glean.pingUploadScheduler?.process()
```

Comment:
I am not sure I understand this comment with the line below. Is the comment still up-to-date?

Reply:
I can clarify what's going on here in the comment